### PR TITLE
[SRBT-202] fix(add-widgets): adding svgs

### PR DESCRIPTION
**Pull request by audreyostrom**
# [SRBT-202] fix(add-widgets): adding svgs

**Changes**

- Supporting svgs for user widgets

# Screenshots

Screenshots/videos of the changes made (if any, otherwise delete this section)

### CODE REVIEW

File: src/components/profile/widgets/widget-grid.tsx
Comment (ChristopherPark3) [inline src/components/profile/widgets/widget-grid.tsx:1]: Is there a reason for ‘image && image !== undefined’ on line 166? If you didnt write it, can you test to see if the truthy check of ‘image’ suffices?

**Review by ChristopherPark3 — APPROVED** [review]
Nice work!

File: src/components/profile/widgets/widget-grid.tsx
Comment (Cutaiar) [reply src/components/profile/widgets/widget-grid.tsx:1]: Guessing it is checking for when image is null. So agreed on the falsey thing

**Review by Cutaiar — APPROVED** [review]
Approved with suggestions 🚀

File: src/components/profile/widgets/widget-grid.tsx [hunk 1]
```diff
@@ -164,9 +164,15 @@ export const WidgetGrid: React.FC<WidgetGridProps> = ({
 
       // We're attempting to add a photo widget
       if (type === 'Photo' && image && image !== undefined) {
+        const fileExtension = image.name.split('.').pop()?.toLowerCase();
         const imageFormData = new FormData();
         imageFormData.append('file', image);
-        imageFormData.append('fileType', 'image');
+        // fileType must be different for svgs to render correctly
+        if (fileExtension == 'svg') {
```
[the remark this answers is not in the document]
Comment (audreyostrom) [reply src/components/profile/widgets/widget-grid.tsx:171]: That was an oversight on my part!

**Pull request merged**